sdk/formrecognizer/Azure.AI.FormRecognizer/tests/DocumentModelAdministrationClient/DocumentModelAdministrationClientLiveTests.cs

```diff
@@ -123,7 +123,7 @@ public async Task GetPrebuildModel()
         {
             var client = CreateDocumentModelAdministrationClient();
 
-            DocumentModel model = await client.GetModelAsync("prebuilt-receipt");
+            DocumentModel model = await client.GetModelAsync("prebuilt-businessCard");
```

Comment:
Not sure if I'm missing something, but is there a reason for this change?

Reply:
prebuilt-receipt still brings `Currency` and makes our SDK mad and breaks. So I just changed it to another prebuilt.
I have considered making a modification into our deserialization code to adjust for the `Currency` thing, but the deployment with the fix is already rolling and according to Paul it will hit the weekend, so it will be in place before we do our release so we should be ok.

What do you think?

Reply:
Sounds good. If it's not fixed by Monday, we can add the workaround you suggested.